test/components/entity/list.spec.js

```diff
@@ -88,8 +88,8 @@ describe('EntityList', () => {
       let initialTime;
       testData.extendedDatasets.createPast(1, { name: 'trees' });
       testData.extendedEntities.createPast(1);
-      const assertRowCount = (count, responseIndex = 0) => (component, _, i) => {
-        if (i === responseIndex) {
+      const assertRowCount = (count) => (component, _, i) => {
+        if (i === 0 || i == null) {
```

Comment:
I made a parallel change in #1335: see https://github.com/getodk/central-frontend/pull/1335#discussion_r2324171429.